blueprints/app/files/package.json

```diff
@@ -12,10 +12,12 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
```

Comment:
Checkout the [docs on the npm-run-all command](https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/npm-run-all.md), but this is basically running `lint:js` and `lint:hbs` in parallel and ensuring the console output is aggregated (so they don't get interleaved).